src/app/users/users-update/users-update.component.ts

```diff
@@ -95,6 +104,10 @@ export class UsersUpdateComponent implements OnInit {
     this.location.back();
   }
 
+  goBack() {
```

Comment:
goback() and cancel() above it should be one function and it should take to profile view